resources.yaml

```diff
@@ -299,3 +298,10 @@ deployment:
     group: training-joca-epigenomics-24
     image: htcondor-secondary
     secondary_htcondor_cluster: true
+  training-rock:
+    count: 1
+    flavor: c1.c28m225d50
+    start: 2024-04-09
```

Comment:
```suggestion
    start: 2024-03-22
```

Reply:
no that was intentional, I thought there was no need to change the start date

Reply:
But your start date is in April.